synapse/storage/databases/main/purge_events.py

```diff
@@ -441,6 +442,10 @@ def _purge_room_txn(self, txn: LoggingTransaction, room_id: str) -> List[int]:
             "e2e_room_keys",
             "event_push_summary",
             "pusher_throttle",
+            "insertion_events",
+            "insertion_event_extremities",
+            "insertion_event_edges",
+            "batch_events",
```

Comment:
Just asking the question in case, Is there a reason why we wouldn't want to purge related data for a room when deleting the room?

Reply:
Maybe for keeping some kind of audit logs, maybe? Or perhaps you want to kick everyone for a room but keep its content, e.g. to determine what information has been leaked? (Pure speculation)